android/build.gradle

```diff
@@ -27,6 +27,7 @@ def DEFAULT_MIN_SDK_VERSION = 16
 def DEFAULT_TARGET_SDK_VERSION = 31
 
 android {
+  namespace = "com.proyecto26.inappbrowser"
```

Comment:
can you verify the gradle plugin version to have backward compatability please? e.g:
```gradle
def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION.tokenize('.')[0].toInteger()
if (agpVersion >= 7) {
  namespace 'com.proyecto26.inappbrowser'
}
```

Reply:
More examples here:
- https://github.com/th3rdwave/react-native-safe-area-context/blob/main/android/build.gradle#L43
- https://github.com/react-native-netinfo/react-native-netinfo/blob/master/android/build.gradle#L34
- https://github.com/invertase/react-native-firebase/blob/main/packages/app/android/build.gradle#L62